glusto/main.py

```diff
@@ -225,7 +225,7 @@ def main():
     retcode = 0
     if args.run_pytest:
         print "pytest: %s" % args.run_pytest
-        result = pytest.main(args.run_pytest)
+        result = pytest.main(args.run_pytest.split('/'))
```

Comment:
Hey Nigel,
PyTest args need to split on (' ') instead of ('/'), similar to Nose and PyUnit. I already have this fixed in the Python3 branch I'm testing now and intended to push it with a backport of changes that apply to Python2. Can you remove this file, so I can merge the .gitignore without a rebase of the Python3 and backport branches?
Cheers, Jonathan